cirq/contrib/routing/initialization.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2019 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import itertools
+import random
+from typing import cast, Dict, Hashable, List, Set
+
+import networkx as nx
+
+from cirq import ops
+
+
+def get_center(graph: nx.Graph) -> Hashable:
+    centralities = nx.betweenness_centrality(graph)
+    return max(centralities, key=centralities.get)
+
+
+def get_initial_mapping(logical_graph: nx.Graph,
+                        device_graph: nx.Graph) -> Dict[ops.Qid, ops.Qid]:
+    """Gets an initial mapping of logical to physical qubits for routing.
```

Comment:
add a brief description of the algorithm used here

Reply:
Done. (Also fixed some bugs.)